as2/server/player.md

```diff
@@ -0,0 +1,98 @@
+# Player
+##### Messages pertaining to the player actions.
```

Comment:
Should be "pertaining to actions performed by penguin avatars" (or something similiar) as that is much more clear.